### PR TITLE
Update remove-licenses-from-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added clarity to the 30days mentioned in the doc

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/954

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by Ajayan1008** [comment]
@JohanFreelancer9 - Please post here once you've completed your review so we can merge the pull request. Thanks.

File: Office365-Admin/manage/remove-licenses-from-users.md [hunk 1]
```diff
@@ -146,12 +146,14 @@ When you use the **Licenses** page to unassign licenses, you unassign licenses f
...
 
    Also, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or Advanced eDiscovery when the license is removed. For more information, see the "Searching disconnected or de-licensed mailboxes" section in [Content Search in Office 365](https://docs.microsoft.com/office365/securitycompliance/content-search#searching-disconnected-or-de-licensed-mailboxes).
 
 - If you have an Enterprise subscription, like Office 365 Enterprise E3, Exchange Online lets you preserve the mailbox data of a deleted user account by using [inactive mailboxes](https://docs.microsoft.com/office365/securitycompliance/inactive-mailboxes-in-office-365). For more information, see [Create and manage inactive mailboxes in Exchange Online](https://docs.microsoft.com/office365/securitycompliance/create-and-manage-inactive-mailboxes).
 
+- For information about OneDrive deletion process, See [OneDrive retention and deletion](https://docs.microsoft.com/en-us/onedrive/retention-and-deletion).
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/remove-licenses-from-users.md]:
```suggestion
- For information about the OneDrive deletion process, see [OneDrive retention and deletion](https://docs.microsoft.com/onedrive/retention-and-deletion).
```

**Comment by mypil** [comment]
@Ajayan1008 @cmcatee-MSFT - PR has been copy edited and is ready for final review. This is based  on issue #954.

Thank you.

cc: @kenwith

**Pull request merged**